plugin/evm/vm_test.go

```diff
@@ -405,6 +405,7 @@ func TestVMConfigDefaults(t *testing.T) {
 	vmConfig.SetDefaults()
 	vmConfig.RPCTxFeeCap = txFeeCap
 	vmConfig.EnabledEthAPIs = enabledEthAPIs
+	vmConfig.UpdateWithDefaults()
```

Comment:
I can include a call to `UpdateWithDefaults()` at the bottom of the body of `SetDefaults()` if you prefer.